plugin/src/main/kotlin/tanvd/grazi/EnglishChonker.kt

```diff
@@ -0,0 +1,44 @@
+package tanvd.grazi
+
+import com.intellij.openapi.application.ApplicationManager
+import org.jetbrains.org.objectweb.asm.*
+import org.jetbrains.org.objectweb.asm.Opcodes.ASM5
+
+
+class EnglishChonker(cv: ClassVisitor) : ClassVisitor(ASM5, cv) {
+    companion object {
+        fun reloadEnglish() {
+            val stream = GraziPlugin::class.java.getResourceAsStream("/org/languagetool/language/English.class")
```

Comment:
Hm, should not we do it throught some interface , like `Resources.stream(filePath)`? To fix classloader for resources resolution

Reply:
Create our Resources object